Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -1192,6 +1194,25 @@ static bool canSkipComputeCompositingRequirementsForSubtree(const RenderLayer& l
     return layer.renderer().hasNonVisibleOverflow();
 }
 
+bool RenderLayerCompositor::allowsFixedPositionBackingStoreDetaching(RenderLayer& layer, const LayoutRect& absoluteBounds)
+{
+    if (!layer.behavesAsFixed())
```

Comment:
Maybe it should be an error to call this on a non-fixed layer? Otherwise the return value might imply "never detach".

Or perhaps it should be `allowBackingStoreDetachingIfFixedPosition` but some ambiguity remains.